src/api/transfers/routes.js

```diff
@@ -110,17 +110,17 @@ module.exports = [{
       params: {
```

Comment:
Just a question, do you not need to wrap this in a Joi.object as per the payload? I needed to do this in central-ledger